Makefile

```diff
@@ -101,7 +101,7 @@ mongorestore-nmdc-db:
 	mkdir -p /tmp/remote-mongodump/nmdc
 	# SSH into the remote server, stream the dump directory as a gzipped tar archive, and extract it locally.
 	ssh -i ~/.ssh/nersc ${NERSC_USERNAME}@dtn01.nersc.gov \
-		'tar -czf - -C /global/cfs/projectdirs/m3408/nmdc-mongodumps/dump_nmdc-prod_2024-07-29_20-12-07/nmdc .' \
+		'tar -czf - -C /global/cfs/projectdirs/m3408/nmdc-mongodumps/dump_nmdc-prod_2024-11-25_20-12-02/nmdc .' \
```

Comment:
FYI, @dwinston, as of about 24 hours ago (via [this PR](https://github.com/microbiomedata/infra-admin/pull/142)), the daily dumps are being created with the `--gzip` CLI option to `mongodump`. Before that, the `--gzip` CLI option was not being included. So, the `mongorestore` command here may require a change (e.g. the addition of _its_ `--gzip` CLI option—assuming you're restoring a dump that was created after about 24 hours ago).